packages/grid/_modules_/grid/models/api/csvExportApi.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * The csv export API interface that is available in the grid [[apiRef]].
+ */
+export interface CsvExportApi {
+  /**
+   * Export the grid data as CSV.
+   * @returns void
+   */
+  exportDataAsCsv: () => void;
+  /**
+   * Get the grid data as CSV.
+   * @returns
+   */
+  getDataAsCsv: () => any;
```

Comment:
it seems to be a string

```suggestion
  getDataAsCsv: () => string;
```

Reply:
updated